src/components/CellLineInfoCard/CellLineInfoCardBase.tsx

```diff
@@ -127,6 +125,15 @@ const CellLineInfoCard = ({
                 >
                     hPSCreg Certificate
                 </DefaultButton>
+                {orderPlasmid && (
+                    <DefaultButton
+                        href={orderPlasmid}
+                        target="_blank"
+                        rel="noreferrer"
+                    >
+                        Obtain Plasmid
+                    </DefaultButton>
```

Comment:
if I was trying to make this a generic component, I would make a `buttonList: {label: string, href: href}` and then this component doesn't need to know about props like `orderPlasmid`